docker-compose.test.yml

```diff
@@ -7,6 +7,7 @@ services:
     command: ["go", "test", "-v", "github.com/appcelerator/amp/data/etcd"]
```

Comment:
@generalhenry @chrisccoy: `sut` shouldn't be `amp/data/etcd` for this test. There are one of two choices: 1) The reviewer updates the test compose file for a PR or 2) invoke `make test` which should execute all the project tests. I don't like the first choice. With the second, we could call a new rule, like `make test-docker` that exports environment variables and runs the tests. You can supply environment variables to make ([ref](https://www.gnu.org/savannah-checkouts/gnu/make/manual/html_node/Environment.html), but this isn't ideal, and requires updating the compose file environment for sut.